damnit/gui/plot.py

```diff
@@ -423,6 +423,10 @@ def __init__(self, parent, data, **kwargs):
             self._nan_warning_label.show()
 
         data.plot(ax=self._axis)
+
+        if data.ndim == 1:
```

Comment:
I don't think you need the ndim check here - this class is specific to 1D dataarrays, and others will go into `ImagePlotWindow`.

Reply:
Good point, removed in b799785.